cmd/dlb_plugin/README.md

```diff
@@ -34,8 +34,8 @@ If you configure SR-IOV/VF (virtual functions), continue the following configura
 Patch dpdk sources to work with DLB:
 ```bash
 $ wget -q https://fast.dpdk.org/rel/dpdk-20.11.4.tar.xz -O- | tar -Jx
```

Comment:
Maybe not your mistake, but I guess this should be also changed?

Reply:
Thank you for pointing out to this. Should be fixed now.